molecule/default/destroy.yml

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
if I remove this file and run `molecule destroy` I get:

```
INFO     Running default > destroy
WARNING  Skipping, destroy action has no playbook.
```

I think they are mean to be there, even if unmodified (probably because there isn't a way to override them - yet)

Reply:
ok... that's surprising given none of the other tests share this requirement. Something to do with ec2 module in particular?
Well let's be pragmatic for now and keep this playbook as it is for now.